kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -223,6 +227,27 @@
     watch: {
       username: 'setSuggestionTerm',
     },
+    mounted() {
+      /*
+        Chrome has non-standard behavior with auto-filled text fields where
+        the value shows up as an empty string even though there is text in
+        the field:
+          https://bugs.chromium.org/p/chromium/issues/detail?id=669724
+
+        As super-brittle hack to detect the presence of auto-filled text and
+        work-around it, we look for a change in background color as described
+        here:
+          https://stackoverflow.com/a/35783761
+      */
+      setTimeout(() => {
+        const bgColor = window.getComputedStyle(this.$refs.password.$el.querySelector('input'))
+          .backgroundColor;
+        if (bgColor === 'rgb(250, 255, 189)') {
```

Comment:
@indirectlylit I think the reason why this didn't work when @radinamatic tested this is because I tried the `Empty Cache and Hard Reload` the background color returns `rgba(0,0,0,0)`. After refreshing it, it becomes `rgb(250, 255, 189)`